tests/UnitTests.runsettings

```diff
@@ -20,7 +20,7 @@
                     <!-- See https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/VSTestIntegration.md#advanced-options-supported-via-runsettings
                      and https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/MSBuildIntegration.md#excluding-from-coverage -->
                     <Exclude>[NewRelic.OpenTracing.AmazonLambda*]*</Exclude>
-                    <ExcludeByFile>**/tests/**/*, **/NativeMethods.cs</ExcludeByFile>
+                    <ExcludeByFile>**/tests/**/*, **/NativeMethods.cs, **/NewRelic.Api.Agent/NewRelic.cs</ExcludeByFile>
```

Comment:
We might want to add AgentManager.cs to this list too.

Reply:
Done!